src/Entity/Adherent.php

```diff
@@ -1102,8 +1102,7 @@ public function setReferent(array $tags, string $markerLatitude = null, string $
 
     public function isReferent(): bool
     {
-        return $this->managedArea instanceof ReferentManagedArea
-            && !$this->managedArea->getTags()->isEmpty();
+        return $this->managedArea instanceof ReferentManagedArea;
```

Comment:
i remember, a while ago, this extra check missing made the application send an email to the whole database :)
are we sure we want to remove this check ?
is it for performance?
what if we forget to apply this eventlistener on a form one day?

Reply:
same modification/questions would apply to : 
https://github.com/EnMarche/en-marche.fr/blob/master/src/Entity/Adherent.php#L1149
https://github.com/EnMarche/en-marche.fr/blob/master/src/Entity/Adherent.php#L1151

etc etc

i mostly see these checks as a security if we ever have a corrupted entry in the database.

Reply:
agree, I prefer to rollback this part